### PR TITLE
Apply some changes to the manual topics cloud tutorial

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: docs/rst/tutorials/cloud/router_max_tx_rate/router_max_tx_rate.rst [hunk 1]
```diff
@@ -2,8 +2,15 @@
...
-ROS 2 Router configuration with Manual Topics
-=============================================
+.. NOT-YET-AVAILABLE-TUTORIAL
+
+.. warning::
+
+    The |rosrouter| features described in this tutorial are not yet available in the current release of Vulcanexus.
+    Stay tuned for new Vulcanexus releases to start using these new features!
+
+Configuring DDS Topics in the ROS 2 Router
```
Comment (Tempate) [inline docs/rst/tutorials/cloud/router_max_tx_rate/router_max_tx_rate.rst:12]: I think the title of the tutorial is too broad. Maybe `Configuring DDS Topic QoS in the ROS 2 Router` would be more descriptive?

File: docs/rst/tutorials/cloud/router_max_tx_rate/router_max_tx_rate.rst
Comment (Tempate) [inline docs/rst/tutorials/cloud/router_max_tx_rate/router_max_tx_rate.rst:1]: I find the file name `router_max_tx_rate.rst` a bit misleading. Maybe `router_topic_qos.rst` would be better? In any case, please make sure to rename `/rst/figures/tutorials/cloud/change_domain_manual_topics.png` and `/resources/tutorials/cloud/router_conf_with_manual_topics/router_conf_with_manual_topics.yaml` accordingly.

**Review by Tempate — CHANGES_REQUESTED** [review]

File: docs/rst/tutorials/cloud/router_max_tx_rate/router_max_tx_rate.rst (hunk 1 above)
Comment (rsanchez15) [reply docs/rst/tutorials/cloud/router_max_tx_rate/router_max_tx_rate.rst:12]: Update to "Configuring ROS 2 Router's Topic QoS"

**Review by Tempate — APPROVED** [review]
LGTM

**Comment by rsanchez15** [comment]
@Mergifyio backport humble

**Pull request merged**